apps/k8s-io/configmap-nginx.yaml

```diff
@@ -54,6 +54,11 @@ data:
             ';
           }
 
+          # Redirect go import paths to Go package documentation.
+          if ($repo ~ "api|apimachinery|client-go|kubernetes") {
```

Comment:
I think I'd implement this kind of redirect by editing https://github.com/kubernetes/website/blob/main/static/_redirects

Would that work? It's more visible and less likely to conflict with SIG Docs changes / plans.

Reply:
Ooh, nice. I didn't know that was also an option, this definitely seems like an easier place to put these, with preview deploys and tests and everything. I'll look into it, and possibly close this PR. 👍

Reply:
Yeah, I think this might be the better overall path as then we are saying: "unless we know for sure they are trying to do a `go get`, then it should go to the website, and then sig-docs can do things there. Especially if this is a manual list.